src/pretix/control/templates/pretixcontrol/checkin/index.html

```diff
@@ -74,6 +76,11 @@
                     {% for e in entries %}
                         {% with e.checkins.first as checkin %}
                         <tr>
+                          <td> 
+                            <input type="checkbox" name="checkin"
+                                                   id="id_checkin" class=""
+                                                                   value="{{e.order.code}}"/>
```

Comment:
This won't work: Imagine you buy two tickets with the same order: They will turn up as two rows here, but both will have the same code. I think we can just use ``e.pk`` here.